### PR TITLE
minor doc improvements

### PR DESCRIPTION
**Pull request by jugmac00**
@gaborbernat No need to review right now - I will go through the changes once more tonight. I am a bit in a hurry right now and the diff in the GitHub UI is actually better than the one on my local cli :-)

This was a quick reading through the docs - I will do a thorough proofreading again, maybe 2 weeks before a final release.

### CODE REVIEW

**Comment by jugmac00** [comment]
I tried to use  a towncrier file as suggested in the docs a la "something.trivial" but this was not permitted by `pre-commit` :-D

```
changelog filenames......................................................Failed
- hook id: changelogs-rst
- exit code: 1

changelog files must be named ####.(feature|bugfix|doc|removal|misc).rst

docs/changelog/minor.doc.improvements.trival
```

**Comment by gaborbernat** [comment]
> changelog files must be named ####.(feature|bugfix|doc|removal|misc).rst

Update the documentation please per this rule.

File: docs/development.rst [hunk 1]
```diff
@@ -151,22 +151,22 @@ or whitespace within lines. Such changes can be made separately, as a "formattin
 Automated testing
 ~~~~~~~~~~~~~~~~~
 
-All pull requests and merges to 'master' branch are tested using
-`Azure Pipelines <https://azure.microsoft.com/en-gb/services/devops/pipelines/>`_ (configured by
-``azure-pipelines.yml`` file at the root of the repository). You can find the status and results to the CI runs for your
+All pull requests and merges to the ``master`` branch are tested using
+`GitHub Actions <https://github.com/features/actions>`_ (configured by ``check.yml`` file inside the
+``.github/workflows`` directory). You can find the status and results to the CI runs for your
 PR on GitHub's Web UI for the pull request. You can also find links to the CI services' pages for the specific builds in
```
Comment (jugmac00) [inline docs/development.rst:157]:
```suggestion
PR on GitHub's Web UI for the pull request. You can also find links for the CI services' pages for the specific builds in
```

File: docs/development.rst [hunk 2]
```diff
@@ -151,22 +151,22 @@ or whitespace within lines. Such changes can be made separately, as a "formattin
 Automated testing
 ~~~~~~~~~~~~~~~~~
 
-All pull requests and merges to 'master' branch are tested using
-`Azure Pipelines <https://azure.microsoft.com/en-gb/services/devops/pipelines/>`_ (configured by
-``azure-pipelines.yml`` file at the root of the repository). You can find the status and results to the CI runs for your
+All pull requests and merges to the ``master`` branch are tested using
+`GitHub Actions <https://github.com/features/actions>`_ (configured by ``check.yml`` file inside the
+``.github/workflows`` directory). You can find the status and results to the CI runs for your
```
Comment (jugmac00) [inline docs/development.rst:156]:
```suggestion
``.github/workflows`` directory). You can find the status and the results to the CI runs for your
```

File: docs/user_guide.rst [hunk 3]
```diff
@@ -73,10 +74,10 @@ tox roughly follows the following phases:
    change.
 
    3. **commands**: run the specified commands in the specified order. Whenever the exit code of any of them is not
-   zero, stop and mark the environment failed. Note, starting a command with a single dash character means ignore exit
-   code.
+   zero, stop and mark the environment failed. You can ignore the exit code when you start a command with a single dash
+   character.
```
Comment (jugmac00) [inline docs/user_guide.rst:78]:
```suggestion
   zero, stop and mark the environment failed. When you start a command with a dash character, the exit code will be ignored.
```

File: src/tox/plugin/spec.py [hunk 4]
```diff
@@ -20,7 +20,7 @@ def _spec(func: _F) -> _F:
 @_spec
 def tox_register_tox_env(register: ToxEnvRegister) -> None:  # noqa: U100
     """
-    Register new tox environment type that. You can register:
+    Register new tox environment type that you can register:
```
Comment (jugmac00) [inline src/tox/plugin/spec.py:23]:
```suggestion
    Register new tox environment type. You can register:
```

File: docs/development.rst [hunk 5]
```diff
@@ -71,14 +71,14 @@ select tests using the various ways that pytest provides:
     tox -e py -- -k "test_extra"
 
 Some tests require additional dependencies to be run, such is the various shell activators (``bash``, ``fish``,
-``powershell``, etc). These tests will automatically be skipped if these are not present, note however that in CI
+``powershell``, etc). These tests will be skipped automatically if these are not present, note however that in CI
```
Comment (jugmac00) [inline docs/development.rst:74]:
```suggestion
``powershell``, etc). The tests will be skipped automatically if the dependencies are not present, note however that in CI
```

File: docs/development.rst [hunk 6]
```diff
@@ -71,14 +71,14 @@ select tests using the various ways that pytest provides:
     tox -e py -- -k "test_extra"
 
 Some tests require additional dependencies to be run, such is the various shell activators (``bash``, ``fish``,
-``powershell``, etc). These tests will automatically be skipped if these are not present, note however that in CI
+``powershell``, etc). The tests will be skipped automatically if the dependencies are not present, note however that in CI
```
Comment (jugmac00) [inline docs/development.rst:74]:
```suggestion
``powershell``, etc). The tests will be skipped automatically if the dependencies are not present. Please note however that in CI
```

File: docs/development.rst [hunk 7]
```diff
@@ -183,27 +183,27 @@ added a feature and deprecated/removed the old feature at the same time, you wou
 ``docs/changelog/1234.bugfix.rst`` and ``docs/changelog/1234.remove.rst``). Likewise if a PR touches multiple issues/PRs
 you may create a file for each of them with the same contents and :pypi:`towncrier` will deduplicate them.
 
-Contents of a NEWS entry
-^^^^^^^^^^^^^^^^^^^^^^^^
+Contents of a changelog entry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The contents of this file are reStructuredText formatted text that will be used as the content of the news file entry.
+The contents of this file are reStructuredText formatted text that will be used as the content of the changelog entry.
```
Comment (jugmac00) [inline docs/development.rst:189]:
```suggestion
The content of this file is reStructuredText formatted text that will be used as the content of the changelog entry.
```

File: docs/development.rst [hunk 8]
```diff
@@ -183,27 +183,27 @@ added a feature and deprecated/removed the old feature at the same time, you wou
...
 by a blank line separating it from a description of the feature/change in one or more paragraphs, each wrapped
-at 120 characters. Remember that a news entry is meant for end users and should only contain details relevant to an end
-user.
+at 120 characters. Remember that a changelog entry is meant for end users and should only contain details relevant to an
+end user.
 
-Choosing the type of NEWS entry
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Choosing the type of changelog entries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (jugmac00) [inline docs/development.rst:202]:
```suggestion
```

File: docs/development.rst [hunk 9]
```diff
@@ -183,27 +183,23 @@ added a feature and deprecated/removed the old feature at the same time, you wou
...
-at 120 characters. Remember that a news entry is meant for end users and should only contain details relevant to an end
-user.
-
-Choosing the type of NEWS entry
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+at 120 characters. Remember that a changelog entry is meant for end users and should only contain details relevant to an
+end user.
 
-A trivial change is anything that does not warrant an entry in the news file. Some examples are: code refactors that
 don't change anything as far as the public is concerned, typo fixes, white space modification, etc. To mark a PR
```
Comment (jugmac00) [inline docs/development.rst:200]:
```suggestion
```

File: docs/development.rst [hunk 10]
```diff
@@ -183,27 +183,22 @@ added a feature and deprecated/removed the old feature at the same time, you wou
...
-Choosing the type of NEWS entry
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+at 120 characters. Remember that a changelog entry is meant for end users and should only contain details relevant to an
+end user.
 
-A trivial change is anything that does not warrant an entry in the news file. Some examples are: code refactors that
-don't change anything as far as the public is concerned, typo fixes, white space modification, etc. To mark a PR
-as trivial a contributor simply needs to add a randomly named, empty file to the ``news/`` directory with the extension
-of ``.trivial``.
+as trivial a contributor simply needs to add a randomly named, empty file to the ``docs/changelog/`` directory with the
```
Comment (jugmac00) [inline docs/development.rst:200]:
```suggestion
```

File: docs/user_guide.rst [hunk 11]
```diff
@@ -22,15 +23,15 @@ file residing at the root of your project:
         # NOTE: you can run any command line tool here - not just tests
         pytest
 
-You can also try generating a ``tox.ini`` file automatically, by running ``tox-quickstart`` and then answering a few
-simple questions. To sdist-package, install and test your project against Python3.7 and Python3.8, just type:
+You can also try generating a ``tox.ini`` file automatically by running ``tox-quickstart`` and then answering a few
```
Comment (jugmac00) [inline docs/user_guide.rst:26]:
```suggestion
You can also try generating a ``tox.ini`` file automatically by running ``tox quickstart`` and then answering a few
```

**Comment by jugmac00** [comment]
@gaborbernat, should be ready to review!

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**